node/src/command.rs

```diff
@@ -451,7 +460,7 @@ pub fn run() -> Result<()> {
 						sp_core::crypto::set_default_ss58_version(
 							amplitude_runtime::SS58Prefix::get().into(),
 						);
-						crate::service::start_parachain_node_spacewalk_amplitude::<AmplitudeRuntimeExecutor>(
```

Comment:
I could remove the generic type parameters from the functions `start_parachain_node_spacewalk_amplitude`, etc.